### DISCUSSION TITLE
Ant JavaFX FXML Application: Execution Error .jar File.

### DISCUSSION THREAD

**Discussion by ACimaroli** [apache/netbeans#3834] [General]
### Apache NetBeans version

Apache NetBeans 13
Windows 11 

### What happened

[manifestmf.txt](https://github.com/apache/netbeans/files/8326704/manifestmf.txt)
[buildxml.txt](https://github.com/apache/netbeans/files/8326703/buildxml.txt)

Ant JavaFX FXML Application

Project Properties - Libraries:
Classpath: javafx.classpath.extension
C:\Program Files\Java\jdk1.8.0_311\jre\lib\ext\jfxrt.jar

Run: OK!
----------

.Jar file execution: Double click in Windows environment does not run.

>java -jar C:\Users\acima\Documents\NetBeansProjects\combobox1\dist\combobox1.jar
Error: JavaFX runtime components are missing, and are required to run this application

I look forward to your suggestions. THANK YOU.

Alberto Cimaroli

a.cimaroli@alice.it

### How to reproduce

[Combobox1.java.txt](https://github.com/apache/netbeans/files/8318132/Combobox1.java.txt)
[FXMLDocument.fxml.txt](https://github.com/apache/netbeans/files/8318134/FXMLDocument.fxml.txt)
[FXMLDocumentController.java.txt](https://github.com/apache/netbeans/files/8318138/FXMLDocumentController.java.txt)
![tigre](https://user-images.githubusercontent.com/99753824/159335159-f70b23f9-322f-417e-909f-b426227cbc10.jpg)
![leone](https://user-images.githubusercontent.com/99753824/159336286-dce7c8af-a03d-4842-aa4a-e81c8ac7e057.jpg)
![lupo](https://user-images.githubusercontent.com/99753824/159335149-188c8095-0f3d-4936-8226-292ddba1dc14.jpg)

### Did this work correctly in an earlier version?

No

### Operating System

Windows 11

### JDK

JDK 1.8

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by mbien** [comment]
English please. And please attach log files instead of pasting it into description fields.

**Comment by ACimaroli** [comment]
> I have completed the information as suggested by you.
> Thank you.

**Comment by neilcsmith-net** [comment]
This looks like a usage question (and possibly not even NetBeans specific) to me.  Converting to discussion.
